addon/components/form-validator.ts

```diff
@@ -0,0 +1,59 @@
+import Component from '@ember/component';
+// @ts-ignore: Ignore import of compiled template
+import layout from '../templates/components/form-validator';
+import { tagName } from '@ember-decorators/component';
+import { A } from '@ember/array';
+import NativeArray from '@ember/array/-private/native-array';
+import { isNone, tryInvoke } from '@ember/utils';
+import { resolve, reject, all } from 'rsvp';
+import { action } from '@ember-decorators/object';
+
+@tagName('form')
+export default class FormValidator extends Component{
+    layout = layout;
+    didInvokeValidate: boolean = false;
+    childValidators: NativeArray<{}> = A();
+    changeset: any;
+    showAllValidationFields: boolean = false;
+
+    registerChild(formValidator: any) {
```

Comment:
is it possible to import the `form-validator/child` component and change this type declaration to `formValidator: FormValidatorChild`?